apps/frontend/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "martletplace-frontend",
+  "private": true,
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "start": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "format": "prettier --write ./src",
+    "format:ci": "prettier --check ./src",
+    "test": "vitest",
+    "preview": "vite preview"
+  },
+  "dependencies": {
```

Comment:
This will dev and regular dependencies will need:
- axios
- react-router-dom
- cypress
- @cypress/code-coverage
- @mui/material 
- @emotion/react 
- @emotion/styled
- vite-plugin-istanbul

Reply:
Yep, those can be added when you start needing them. This PR is just to template out the most basic frontend possible (trying to keep PRs extremely small)